indico/web/flask/session.py

```diff
@@ -229,6 +229,12 @@ def save_session(self, app, session, response):
             # session contents so we need to add a `Vary: Cookie`` header
             response.vary.add('Cookie')
 
+        _non_session_refreshing_endpoints = {
+            'core.session_expiration'
```

Comment:
I think this should go into `should_refresh_session` (together with the regex i suggested in the comment on the other PR)

Reply:
Okay, I'll add this